adapters/huaweiads/huaweiads.go

```diff
@@ -432,13 +441,12 @@ func getHeaders(huaweiAdsImpExt *openrtb_ext.ExtImpHuaweiAds, request *openrtb2.
 }
 
 // getHuaweiAdsReqJson: get body json for HuaweiAds request
-func getHuaweiAdsReqJson(request *huaweiAdsRequest, openRTBRequest *openrtb2.BidRequest,
-	huaweiAdsImpExt *openrtb_ext.ExtImpHuaweiAds, extraInfo ExtraInfo) (countryCode string, err error) {
+func getHuaweiAdsReqJson(request *huaweiAdsRequest, openRTBRequest *openrtb2.BidRequest, extraInfo ExtraInfo) (countryCode string, err error) {
```

Comment:
NItpick: Could we rename this function to just be `getReqJson()`. Same for the other functions like `getHuaweiAdsReqAppInfo()` and `getHuaweiAdsReqNetworkInfo() `? I think it would be cleaner to omit HuaweiAds from these functions since all of them live in `huaweiads.go`

Reply:
corrected， thank you